DualityEditorPlugins/EditorBase/PropertyEditors/Resources/FontContentPropertyEditor.cs

```diff
@@ -27,7 +27,7 @@ protected override void OnGetValue()
 			Font font = this.GetValue().FirstOrDefault() as Font;
 
 			bool lastCanRenderFont = this.canRenderFont;
-			this.canRenderFont = (font != null && font.EmbeddedTrueTypeFont != null);
+			this.canRenderFont = (font != null && font.GetGlyphBitmap(' ') != null);

```

Comment:
> @ilexp Can you explain what you mean by "Render" in this case? In the case of a bitmap font you will never have a bitmap font, but it can still be rendered (drawn to screen) no?

Ah, allright. That might have caused some confusion. When I speak of rendering a Font in this case, I mean re-rendering the .ttf vector data onto a set of glyph bitmaps. This has nothing to do with rendering a Font ingame, since all Fonts can be rendered in that sense if they contain any usable data.

> The problem I had with the property grid making all properties readonly was directly related to this because without it running, the canRenderFont was set to false and therefore all properties were set to readonly.

Yep, that is a problem for the kerning switch. We'll need to figure this one out separately.